chia/full_node/mempool_manager.py

```diff
@@ -520,7 +520,7 @@ async def validate_spend_bundle(
         log.log(
             logging.DEBUG if duration < 2 else logging.WARNING,
             f"add_spendbundle {spend_name} took {duration:0.2f} seconds. "
-            f"Cost: {cost} ({round(100.0 * cost/self.constants.MAX_BLOCK_COST_CLVM, 3)}% of max block cost)",
+            f"Cost: {cost} ({round(100.0 * cost/self.max_block_clvm_cost, 3)}% of max block cost)",
```

Comment:
I'm not sold on this change. I think it still makes sense to log the fill rate relative to the consensus rules

Reply:
Yes, I could go either way here. Or even tell them both (X% of consensus, Y% of YOUR configuration).